torch/distributed/optim/zero_redundancy_optimizer.pyi

```diff
@@ -19,9 +19,10 @@ class _DDPBucketAssignment:
     tensor: torch.Tensor | None
 
 class _OverlapStatus(enum.IntEnum):
-    UNINITIALIZED: int = ...
-    DDP_HAS_REBUILT_BUCKETS: int = ...
-    INITIALIZED: int = ...
+    # Ignores needed to avoid mypy error 'enum members must be left unannotated'
+    UNINITIALIZED: int = ... # type: ignore[misc]
+    DDP_HAS_REBUILT_BUCKETS: int = ... # type: ignore[misc]
+    INITIALIZED: int = ... # type: ignore[misc]
```

Comment:
```suggestion
    UNINITIALIZED
    DDP_HAS_REBUILT_BUCKETS
    INITIALIZED
```
I think this is the proper way to fix it, but I haven't tested it.

Reply:
Actually not sure if these are suppose to have the `...` or not.

Reply:
turns out mypy remains happy with the `...` left in